airflow-core/pyproject.toml

```diff
@@ -134,7 +134,7 @@ dependencies = [
     # Does not work with it Tracked in https://github.com/fsspec/universal_pathlib/issues/276
     "universal-pathlib>=0.2.2,!=0.2.4",
     "uuid6>=2024.7.10",
-    "apache-airflow-task-sdk<1.1.0,>=1.0.0",
+    "apache-airflow-task-sdk<=1.1.0.dev0,>=1.0.0",
```

Comment:
As discussed on slack - let's get rid of the dev0 hard-coding  like https://github.com/apache/airflow/pull/49621

Reply:
This will simplify things

Reply:
```suggestion
    "apache-airflow-task-sdk<=1.1.0,>=1.0.0",
```